examples/ota-requestor-app/linux/main.cpp

```diff
@@ -90,7 +97,7 @@ static void InitOTARequestor(void)
     // TODO: instatiate and initialize these values when QueryImageResponse tells us an image is available
```

Comment:
I think these two TODOs can actually be removed. For one, we can indicate where to store a file once it has been downloaded prior a query actually occuring. And two, it doesn't make sense for the QueryImageResponse to indicate where the file should be saved once downloaded (that should be application specific which is what you are doing in this PR).